EIPS/eip-xtndr-family.md

```diff
@@ -0,0 +1,98 @@
+# EIP: XTNDR opcode family
+
+title: XTNDR family opcode
+description: Opcodespace extension via prefix, variadic opcodes and bytecode compression through macros
+author: Brayton Goodall, Mihir Faujdar
+type: Standards Track
+category: Core
+created: 2021-11-30
+
```

Comment:
```suggestion
---
eip: 4515
title: XTNDR family opcode
description: Opcodespace extension via prefix, variadic opcodes and bytecode compression through macros
author: Brayton Goodall (@???), Mihir Faujdar (@???)
discussions-to: <URL>
status: Draft
type: Standards Track
category: Core
created: 2021-11-30
---
```
Please fill in the author GitHub handles and create a discussion thread at https://ethereum-magicians.org/ and paste a link in the `discussions-to` field.